compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -402,20 +411,84 @@ internal class ComposeSceneMediator(
         view.embedSubview(userInputView)
     }
 
+    private var lastFocusedRect: Rect? = null
+    private fun getFocusedRect(): Rect? {
+        return scene.focusManager.getFocusRect()?.also {
+            lastFocusedRect = it
+        } ?: lastFocusedRect
+    }
+
+    @OptIn(ExperimentalComposeApi::class)
     fun setContent(content: @Composable () -> Unit) {
         view.runOnceOnAppeared {
             focusStack?.pushAndFocus(userInputView)
 
             scene.setContent {
                 ProvideComposeSceneMediatorCompositionLocals {
-                    interopContainer.TrackInteropPlacementContainer(
-                        content = content
-                    )
+                    if (configuration.onFocusBehavior == OnFocusBehavior.FocusableAboveKeyboard) {
```

Comment:
Please extract it into a separate, clear-named function. `setContent` should delegate such details

Reply:
Done